### PR TITLE
Sidebar logic with homepage

### PR DESCRIPTION
**Pull request by DimitarIM**
No images yet / Waiting for Adam's homepage structure

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    length: "65cm",
+    weight: 900/1000,
+    habitat: "SE Australia"
+},
```
Comment (Alexander-Lindstroem) [inline js/index.js]: The array of objects should be made using a constructor

**Review by Alexander-Lindstroem — CHANGES_REQUESTED** [review]

File: js/index.js (hunk 1 above)
Comment (DimitarIM) [reply js/index.js]: Fixed it

**Review by Alexander-Lindstroem — APPROVED** [review]

**Review by Alibaba000101 — COMMENTED** [review]
lGTM

**Review by Alibaba000101 — APPROVED** [review]
Look Ok

**Review by andre84o — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -29,89 +30,99 @@ <h1>Australian Zoo</h1>
...
+                        </a>
+                        <span class="tooltip">Echidna</span>
+                    </li>
+                    <li>
+                        <a class="animal" href="#">
+                            <img src="./icons/tasmanian-devil.png" alt="tasmanian-devil Icon"
+                                class="icon-side tasmanian-devil -icon">
+                            <span class="nav-item">Tasmanian Devil</span>
+                        </a>
+                        <span class="tooltip">Tasmanian Devil</span>
```
Comment (Alexander-Lindstroem) [inline index.html:48]: You still need to generate the sidebar HTML from JS according to Rob

**Review by Alexander-Lindstroem — CHANGES_REQUESTED** [review]
You still need

**Pull request closed without merging**